packages/components/pie-modal/src/index.ts

```diff
@@ -80,25 +98,23 @@ export class PieModal extends RtlMixin(LitElement) {
         this.isOpen = false;
     };
 
-    connectedCallback () : void {
-        super.connectedCallback();
-        document.addEventListener(ON_MODAL_OPEN_EVENT, this._handleModalOpened.bind(this));
-        document.addEventListener(ON_MODAL_CLOSE_EVENT, this._handleModalClosed.bind(this));
-    }
-
-    disconnectedCallback () : void {
-        document.removeEventListener(ON_MODAL_OPEN_EVENT, this._handleModalOpened.bind(this));
-        document.removeEventListener(ON_MODAL_CLOSE_EVENT, this._handleModalClosed.bind(this));
-        super.disconnectedCallback();
-    }
+    /**
+     * Prevents the user from dismissing the dialog via the `cancel`
+     * event (ESC key) when `isDismissible` is set to false.
+     *
+     * @param event
```

Comment:
do we need some more information for the event param here?

Reply:
I think this one is fairly straight forwards? The `event: Event` which is the `Event` interface passed through via the eventListener?

Reply:
fixed.